### ISSUE TITLE
geo tool services

### ISSUE THREAD

**Issue by GVogeler** [icaruseu/mom-ca#678]
to solve #299 with https://github.com/CTeyson/geotool we need services to extract:
* all geolocations of charters from one collection
* all geolocations of charters from one archive
* all geolocations of charters from one fonds
* all geolocations of archives
* all geolocations from all charters in MOM

**Comment by GVogeler** [comment]
use part of atom:id after `tag:www.monasterium.net,2011:/` as parameter?

**Comment by StephanMa** [comment]
Wenn i would suggest to use the entire atom-id. Substrings, if neccessary, can be Substracted seperately

**Comment by StephanMa** [comment]
When you havent installed an English Keyboard Layout :(

**Comment by JBigalke** [comment]
The informations for the geolocations will be all in mom.XRX-data/metadata.geo.public/geo.xml ?

**Comment by StephanMa** [comment]
I guess, you mean `/db/mom-data/metadata.geo.public/geo.xml/` ?

**Comment by JBigalke** [comment]
Yes

**Comment by GVogeler** [comment]
So go along with `/db/mom-data/metadata.geo.public/geo.xml` (see #677 for the suggested format)

**Comment by JBigalke** [comment]
In issue #299 we said we use cei:issued/cei:placeName/@key to link a charter to a geolocations ?
Which field do we use for archives, collections, fonds, usw?

Something like: eag:dsc/eag:geokey or eag:dsc/eag:municipality/@geokey or different?

**Comment by GVogeler** [comment]
1. collections and fonds don't have a geo-location and won't be displayed on a map.

2. For archives we use http://web.archive.org/web/*/http://aer.mcu.es/sgae/jsp/censo_guia/Documentos/EAG.DTD.txt as schema (EAG 0.2), which does not include any structure for georeferencing. EAG 0.2 is replaced by EAG 2012 (http://www.bundesarchiv.de/archivgut_online/standards_werkzeuge/02255/index.html) which offers `eag:location` with `@latitude` and `@longitude`. I've prepared already a file with geo coordinates of the cities, in which the archives are located. Thus I would suggest to use
```
<eag:desc>
  <eag:location latitude="46.62472" longitude="14.30528"/>
  ....
</eag:desc>
```
as data structure, allowing for detailed coordinates (in the city) if provided by archives.

**Comment by GVogeler** [comment]
solved with #866 ?

**Comment by StephanMa** [comment]
Would say so... closing
